head/core/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# This CMakeLists.txt handles compiling all the parts of the head
+# module that are portable between host and cross compilation as a static
+# library. It is included in both host and cross configurations.
+
+add_library(head-core STATIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/
```

Comment:
you can't really do this, you need to specify files. and as you've probably noticed from the failing builds, it needs a file to also determine the language